corehq/apps/app_manager/templates/app_manager/v1/managed_app.html

```diff
@@ -110,12 +110,22 @@
                     <li class="edit-module-li {% ifequal module.id selected_module.id %}{% if not form %}
                         active{% endif %}{% endifequal %}"
                         data-index="{{ module.id }}">
-                        <a href="{% url "view_module" domain app.id module.id %}">
+                        <a href="{% url "view_module" domain app.id module.id %}"
+                            >
                             <i class="drag_handle"></i>
-                            {% if module.module_type == 'advanced' %}<i class="fa fa-flask appmanager-icon-type"></i>
-                            {% elif module.module_type == 'report' %}<i class="fa fa-bar-chart appmanager-icon-type"></i>
-                            {% elif module.module_type == 'shadow' %}<i class="fa fa-folder-open-o appmanager-icon-type"></i>
-                            {% else %}<i class="fa fa-folder-open"></i>{% endif %}
+                            <i
+                            {% if module.module_type == 'advanced' %}class="fa fa-flask appmanager-icon-type"
+                            {% elif module.module_type == 'report' %}class="fa fa-bar-chart appmanager-icon-type"
+                            {% elif module.module_type == 'shadow' %}class="fa fa-folder-open-o appmanager-icon-type"
+                            {% else %}class="fa fa-folder-open"{% endif %}
+
+                            {% if module.case_type and request|toggle_enabled:"SUPPORT" %}
```

Comment:
How about releasing it to everyone, but limiting it to apps that have multiple case types? That'd keep it from being distracting for new users / simple apps.

FWIW, the form icons are going to be released to everyone in app manager v2, we're just holding off for the sake of bundling visual changes together. I'm not worried about that here, though, because this is change you only see on hover.